algorand/Dockerfile.build

```diff
@@ -1,15 +1,16 @@
 FROM docker.io/python:3.10@sha256:eeed7cac682f9274d183f8a7533ee1360a26acb3616aa712b2be7896f80d8c5f as algorand-contracts-build
 
-RUN python3 -m pip install virtualenv
-COPY Pipfile.lock Pipfile.lock
-COPY Pipfile Pipfile
+RUN mkdir algosdk
+RUN --mount=type=cache,target=/algosdk wget https://github.com/algorand/go-algorand/releases/download/v3.19.0-stable/node_stable_linux-amd64_3.19.0.tar.gz --output-document=algosdk/node-stable-linux.tar.gz
```

Comment:
does this cache mount do anything? the file is deleted in the next run anyway, and even if it wasn't, this doesn't provide any better caching than what docker would do anyway by just caching the output of the `RUN` layer without the cache mount. applies to the line below too

Reply:
It probably doesn't. I thought I needed it to cache locally but I realized I was modifying a previous layer afterwards invalidating everything. I'll try removing it and see how it behaves.